src/std_bridge/result.rs

```diff
@@ -44,3 +44,12 @@ pub struct ResultPtrAndPtr {
     pub is_ok: bool,
     pub ok_or_err: *mut std::ffi::c_void,
 }
+
+// Bridges `Result<(), E>` where `E` is able to be encoded
```

Comment:
Mind updating this comment to make it clear that this can also be used for types that don't require any bytes to encode such as `struct UnitStruct;`